src/main/java/com/practicesoftwaretesting/common/asserts/NotFoundResponseAssert.java

```diff
@@ -0,0 +1,20 @@
+package com.practicesoftwaretesting.common.asserts;
+
+import com.practicesoftwaretesting.common.responses.ResponseMessage;
+
+import lombok.AllArgsConstructor;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+@AllArgsConstructor
+public class NotFoundResponseAssert {
```

Comment:
Actually, I don't like the naming of `NotFoundResponseAssert`, what is the idea behind it?
Isn't looks better than `ResponseMessageAssert(messageAfterDelete).userNotFound()`
`NotFoundResponseAssert(messageAfterDelete).userNotFound()`

Let's discuss what is the purpose of the class? To handle assertion of the class `ResponseMessage`, but not of the class `NotFoundResponse`